pow/src/main/java/tech/pegasys/teku/pow/fallback/strategy/Eth1ProviderSelector.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.pow.fallback.strategy;
+
+import tech.pegasys.teku.pow.Eth1Provider;
+
+public interface Eth1ProviderSelector {
```

Comment:
nit: I think I'd scrap this interface since we only have one implementation.  We can always extract an interface later if we wind up with multiple implementations.

Reply:
Ok. I removed the interface